### ISSUE TITLE
Incorrect ties with interleaved voices

### ISSUE THREAD

**Issue by TCPN** [paulrosen/abcjs#277]
When I have multiple voices and write the same line in each voices together (so the voices are interleaved), ties that cross lines will be displayed incorrectly.
This issue occurs both in the case of grouped staves and stacked staves.
```
%% staves (a b)
X: 1
M: 4/4
L: 1/4
Q: 1/4=150
K: C
%1
[V: a] EEEF-|
[V: b] CCCC|
%2
[V: a] FECD|
[V: b] CCCG,-|
%3
[V: a] DDDE|
[V: b] G,G,G,G,|
%4
[V: a] C4|]
[V: b] C4|]
```
![image](https://user-images.githubusercontent.com/9272507/69898128-f3d07f00-138f-11ea-94cf-9faea4f971df.png)
![image](https://user-images.githubusercontent.com/9272507/69898130-00ed6e00-1390-11ea-8e5b-dfb2449111e9.png)

**Comment by paulrosen** [comment]
Thanks for the bug report. I'll do another release this month, hopefully with a fix for this.

**Comment by paulrosen** [comment]
You probably already figured this out -- a work around is to put each voice next to each other in the string:
```
[V: a] EEEF-|
[V: a] FECD|
[V: a] DDDE|
[V: b] CCCC|
[V: b] CCCG,-|
[V: b] G,G,G,G,|
```

**Comment by paulrosen** [comment]
This will be fixed soon in version 5.10.4

**Comment by paulrosen** [comment]
Fixed in version 5.11.0
